workflows/data_pipelines/egapro/config.py

```diff
@@ -10,8 +10,12 @@
     tmp_folder=f"{DataSourceConfig.base_tmp_folder}/egapro",
     minio_path="egapro",
     file_name="egapro",
-    resource_id="d434859f-8d3b-4381-bcdb-ec9200653ae6",
-    url=f"{DATA_GOUV_BASE_URL}d434859f-8d3b-4381-bcdb-ec9200653ae6",
+    files_to_download={
+        "egapro": {
+            "url": f"{DATA_GOUV_BASE_URL}d434859f-8d3b-4381-bcdb-ec9200653ae6",
+            "resource_id": "d434859f-8d3b-4381-bcdb-ec9200653ae6",
+        }
+    },
```

Comment:
I don't quite understand the point of repeating `egapro` inside the dict, as it's  already egapro config.

Reply:
See in https://github.com/annuaire-entreprises-data-gouv-fr/search-infra/pull/441#discussion_r1895532423